### ISSUE TITLE
Zooming is extremely slow and laggy on Safari 14

### ISSUE THREAD

**Issue by Exelord** [excalidraw/excalidraw#2837]
Hey,
recently I was switching to Safari and noticed that the Excalidraw zooming is super laggy. Are you aware of this? Is there anything we could do to improve it?

**Comment by hkdeman** [comment]
I am not sure if I am doing something wrong but the latest docker images seem to be really slow. I tried the latest (sha-5c73c58) and then tried 14 days old one and they were a lot faster.

**Comment by dwelle** [comment]
@hkdeman is this related to zooming? If not, please open another issue.

**Comment by hkdeman** [comment]
> @hkdeman is this related to zooming? If not, please open another issue.

It is also slow for zoom but mainly in collaboration. I can open an issue for that.

**Comment by Mrazator** [comment]
Recently noticed the same on safari v17, though in the same scene it was reproducible on safari v15.

Linking #7237 as possibly related.

**Comment by Mrazator** [comment]
Coming back, since zoom on Firefox feels also super slow. 

Pinging @dwelle

**Comment by Mrazator** [comment]
FF same for touchpad & for the mouse wheel.

**Comment by Mrazator** [comment]
Safari only touchpad, the mouse wheel is fine.

**Comment by dwelle** [comment]
For me FF is slow only on zoom-out because render is generally slow on firefox. The zoom debouncing we do only concerns itself with resolution (canvas cache redrawing) but still re-renders canvas during zoom (which it has to do).

**Comment by Mrazator** [comment]
> For me FF is slow only on zoom-out because render is generally slow on firefox.
Yeah, based on the profiling output, it looks like that's the case for Firefox.

**Comment by dxnxk-0** [comment]
Same here, Safari hyper slow when zooming in/out, super fast with Chrome.
